requirements.txt

```diff
@@ -1,6 +1,7 @@
-requests
-beautifulsoup4
-yaspin
-python-dateutil
-click
-socketIO-client == 0.5.7.2
\ No newline at end of file
+requests~=2.25.1
+beautifulsoup4~=4.9.3
+yaspin~=2.0.0
+python-dateutil~=2.8.1
+click~=8.0.1
+socketIO-client == 0.5.7.2
+PyQt5~=5.15.4
```

Comment:
`ModuleNotFoundError: No module named 'PyQt5.QtWebEngineWidgets'`
we need to add `PyQtWebEngine` in requirements instead of PyQt5

Reply:
Thanks